src/components/card/Card.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useRef } from "react";
+import Link from "next/link";
+import { IconArrowRight } from "hds-react";
+
+import { Item } from "../../types";
+import Text from "../text/Text";
+import Keyword from "../keyword/Keyword";
+import styles from "./card.module.scss";
+
+function Card({ id, title, infoLines, keywords, pre, href, image }: Item) {
+  const linkRef = useRef(null);
+  const downRef = useRef<Date>(null);
+
+  const handleWrapperMouseDown = (e: React.MouseEvent<HTMLElement>) => {
+    downRef.current = new Date();
+  };
+
+  const handleWrapperMouseUp = (e: React.MouseEvent<HTMLElement>) => {
+    const link = linkRef.current;
+    const up = new Date();
+
+    // Invoke a click on link if
+    // 1. The event did not bubble from the link itself
+    // 2. The user is not attempting to select text
+    if (link !== e.target && up.getTime() - downRef.current.getTime() < 200) {
+      link.click();
+    }
+  };
+
+  return (
+    // These listeners allow us to provide a better UX for mouse while not
+    // hampering the UX for keybord or screen reader users.
+    // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
+    <article
+      onMouseDown={handleWrapperMouseDown}
+      onMouseUp={handleWrapperMouseUp}
+      className={styles.card}
+    >
```

Comment:
As the whole `acticle` is considered a link, should it have hover effect for the cursor? (This might just be me, but it feels weird hovering on a big card like this without cursor changing 😅 )

Reply:
You are correct, I'll add this.